zanata-war/src/main/java/org/zanata/adapter/OpenOfficeAdapter.java

```diff
@@ -31,7 +31,7 @@
  */
 public class OpenOfficeAdapter extends OkapiFilterAdapter {
     public OpenOfficeAdapter() {
-        super(prepareFilter(), IdSource.subDocNameAndTextUnitId, true);
```

Comment:
@seanf deprecating them would be excessive, since the validity of `textUnitId` as a value for `resId` is dependent on the filter.